1-js/05-data-types/01-primitives-methods/article.md

```diff
@@ -15,7 +15,7 @@ An object
 : Is capable of storing multiple values as properties.
 Can be created with `{}`, for instance: `{name: "John", age: 30}`. There are other kinds of objects in JavaScript, e.g. functions are objects.
 
-One of the best things about objects is that we can store a function as one of properties:
+One of the best things about objects is that we can store functions within them as properties:
```

Comment:
I'd refrain from saying "within an object", because technically functions are stored "by reference". Was the original wording bad?

Reply:
To me, "store" without "reference" implies a value is being stored. I would say "we can store a reference to a function as one of the properties"

Reply:
The word "reference" is not welcome here, because the reader does not know about references yet.

Is the original wording bad or acceptable?

Reply:
I'll keep the original for consistency, but alter slightly 'we can store a function as one of **its** properties'